package.json

```diff
@@ -20,7 +20,9 @@
     "coverage": "jest --coverage",
     "lint": "eslint client/src && sass-lint -v",
     "js-fix": "eslint client/src --fix",
-    "pattern-lib": "yarn && start-storybook -p 6006"
+    "pattern-lib": "yarn && node .storybook/scripts/runStoryTeller.js && start-storybook -p 6006",
+    "storybook": "yarn pattern-lib",
+    "static-storybook": "yarn && node .storybook/scripts/runStoryTeller.js && build-storybook"
```

Comment:
Would this make more sense as `pattern-lib` and `static-pattern-lib`, to avoid duplication / inconsistency? It seems like `pattern-lib` in this context is being kept for backwards compatibility, and `storybook` is the new standard, but to me the underlying library shouldn't dictate the action.

Reply:
Yeah adding `storybook` was sort of my bias tbh. I kept writing `yarn storybook` and being mad when it didn't work. Happy to change it.

Reply:
In the interest of removing any blockers I've removed the addition of `storybook` and changed to `static-pattern-lib`.